.envrc

```diff
@@ -206,11 +206,10 @@ else
     venv_name="${SENTRY_DEVENV_HOME}/virtualenvs/sentry"
     if ! command -v devenv >/dev/null; then
       die '
-Please install the `devenv` tool:
+Please install the devenv tool:
     https://github.com/getsentry/devenv#install
 '
-    elif [ ! -f "${venv_name}/bin/activate" ]; then
-        devenv sync
+      devenv sync
```

Comment:
this should be outside the `fi` right?

Reply:
ah shit i totally messed up this rebase - one sec

Reply:
today is not my day